.pullapprove.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+
+group_defaults:
+  always_pending:
+    labels:
+      - do-not-merge
+      - needs-rebase
+      - work-in-progress
+  approve_by_comment:
+    enabled: true
+    approve_regex: '^:+1:'
+  author_approval:
+    ignored: true
+
+groups:
+  develop-reviewers:
+    required: 1
+    teams:
```

Comment:
You can, but we'd want to override it for `master` and I'm not sure how to override with an empty list in YAML.